prime-react/src/main/scala/lucuma/react/primereact/DropdownBase.scala

```diff
@@ -62,7 +62,6 @@ object DropdownBase {
     CDropdown
       .value(props.getter)
       .options(props.optionsWithIndex.raw)
-      .optionValue("value")
```

Comment:
Apparently, this option only needs to be used if `SelectItem`s are not being used for the options. I'm not sure if that used to be the case or not. Anyway, if it is used (at least with SelectItems) it seems to work fine if the mouse is used, but if the keyboard is used, the `value` returned in the event is always `null`. This seems like it is a bug in primereact, but this works correctly, so...